src/ai_hawk/job_manager.py

```diff
@@ -513,12 +500,12 @@ def job_tile_to_job(self, job_tile) -> Job:
             logger.warning(f"Failed to extract job ID: {e}", exc_info=True)
 
         try:
-            job.location = job_tile.find_element(By.CLASS_NAME, 'job-card-container__metadata-item').text
+            job.location = job_tile.find_element(By.XPATH, './/span[contains(normalize-space(), " · ")]').text.split(' · ')[-1].strip()
         except NoSuchElementException:
             logger.warning("Job location is missing.")
         
         try:
-            job.apply_method = job_tile.find_element(By.CLASS_NAME, 'job-card-container__apply-method').text
+            job.apply_method = job_tile.find_element(By.XPATH, ".//div[contains(@class, 'job-card-container__job-insight-text') and normalize-space() = 'Easy Apply']").text
```

Comment:
correct this. Also include this bug fix https://github.com/AIHawk-FOSS/Auto_Jobs_Applier_AI_Agent/issues/927

remove equals "Easy Apply", consider state where Applied, or saved previous application, as of now bot does that when it fails to submit.

Reply:
done. Not complete fix of 927. but at least it marks and go